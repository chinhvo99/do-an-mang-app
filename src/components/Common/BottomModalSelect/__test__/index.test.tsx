import React from 'react';
import { EnumComponent, getComponent, basicTest } from '@utils/testHelper';
import BottomModalSelect from '..';

const data = [
  { id: '0', name: 'Mở bán dự án Residences Quy Nhơn' },
  { id: '17', name: 'Công bố dự án Phúc Yên Prosper Phố Đông Thủ Đức' },
  { id: '3', name: 'Công bố dự án Century City Long Thành' },
  { id: '5', name: 'Mở bán dự án Green Dragon City Quảng Ninh' },
  { id: '6', name: 'Mở bán dự án Red Dragon City Hà Nội' },
  { id: '7', name: 'Mở bán dự án Violet Dragon City Hải Phòng' },
  { id: '8', name: 'Mở bán dự án Black Dragon City Lào Cai' },
];
const target = <BottomModalSelect data={data} />;
const type = EnumComponent.SINGLE;

describe('BottomModalSelect', () => {
  basicTest(target, type, 1, false, false);
  it('should apply all custom props', () => {
    const initialTest = getComponent(target, {}, type).props();
    expect(initialTest.style.flex).toBe(1);
  });
});
