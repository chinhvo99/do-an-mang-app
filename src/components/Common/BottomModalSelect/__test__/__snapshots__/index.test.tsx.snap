// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomModalSelect should render without issues 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
  testID="BottomModalSelect"
/>
`;
